src/components/CampaignDetails.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section>
+    <h3> Campaing Title: </h3>
+    <h2>{{ campaign.title }}</h2>
+    <h3> Description: </h3>
+    <h4>{{ campaign.description }}</h4>
+    <h3> Minimum Contribution </h3>
+    <h1>{{ campaign.minContribution }}</h1>
+    <p>There is some information missing and this has no styles </p>
+  </section>
+</template>
+
+<script>
+
+export default {
+  name: 'CampaignDetails',
+  props: ['address'],
+  computed: {
+    campaign() {
+      return this.$store.getters.getSingleCampaign(Number(this.address))
+    }
+  }
+}
+</script>
```

Comment:
Gets address from url, and details from store

Reply:
Very semantic with the getters